modules/core_project_factory/main.tf

```diff
@@ -137,6 +137,12 @@ resource "google_compute_shared_vpc_service_project" "shared_vpc_attachment" {
   ]
 }
 
+resource "google_compute_shared_vpc_host_project" "shared_vpc_host" {
+  count      = var.enable_shared_vpc_host ? 1 : 0
+  project    = google_project.main.project_id
+  depends_on = [google_project.main]
```

Comment:
We should probably be depending on `module.project_services` to make sure that compute api is enabled first?

Reply:
You're right!